docs/src/changelog.md

```diff
@@ -0,0 +1 @@
+../../../CHANGELOG.md
```

Comment:
This looks like a file on your fork:
![image](https://user-images.githubusercontent.com/9404707/199836897-104d350b-8529-4c1b-ae3c-b178e6192549.png)
It should appear as a link:
![image](https://user-images.githubusercontent.com/9404707/199836974-55e56598-de9b-4935-8157-98d05424ef1a.png)
From the root element dir, you can run
```console
ln -s ../../CHANGELOG.md ./docs/src/changelog.md
```

Reply:
Fixed it.